airbyte_cdk/sources/declarative/transformations/add_fields.py

```diff
@@ -128,7 +128,7 @@ def transform(
     ) -> None:
         if config is None:
             config = {}
-        kwargs = {"record": record, "stream_state": stream_state, "stream_slice": stream_slice}
+        kwargs = {"record": record, "stream_slice": stream_slice}
```

Comment:
small nit, but in the docstring of the `AddFields` class, we have some comments about `stream_state` , lets remove those so that is up to date